static/src/stylesheets/module/experiments/_ab-acquisitions-love-boat.scss

```diff
@@ -0,0 +1,203 @@
+.contributions__epic {
+    &.contributions__epic--visual {
+        background: $neutral-8;
+    }
+
+    &.contributions__epic--visual--love {
+        border-top-color: $features-support-3;
+    }
+
+    &.contributions__epic--visual--boat {
+        border-top-color: $news-main-2;
+    }
+}
+
+.contributions__title.contributions__title--visual-epic {
+    @include fs-headline(7);
+    font-weight: bold;
+    margin-bottom: $gs-baseline * 3;
+}
+
+.contributions__title--visual-epic--boat {
+    color: $news-support-2;
+}
+
+.contributions__title--visual-epic {
+    @include fs-headline(7);
+    font-weight: bold;
+    margin-bottom: $gs-baseline * 3;
+}
+
+.contributions__title.contributions__title--visual-epic.contributions__title--visual-epic--step-2--love {
```

Comment:
Haha. Surely you only need this `contributions__title--visual-epic--step-2--love` as that should be specific enough no?